.github/workflows/docker.yml

```diff
@@ -0,0 +1,37 @@
+on:
+  push:
+    branches:
+    - main
+    tags:
+    - '*'
+
+name: Release Docker Image
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - name: Install dependencies
+        run: yarn install --immutable --immutable-cache
+
+      - name: Build Docker Image
+        run: yarn docker build backend -t ghcr.io/${{ github.repository_owner }}/rafiki-backend:${{ github.sha }} -t ghcr.io/${{ github.repository_owner }}/rafiki-backend:latest
```

Comment:
If we start building/pushing for other branches, the `latest` tag should only be used for `main`.
But I think this is good for now, unless we want to not use `latest` on git tags?